### DISCUSSION TITLE
how can i use DiscordSRV as a Dependency?

### DISCUSSION THREAD

**Discussion by Outfit8TSB** [DiscordSRV/DiscordSRV#1733] [f. Questions] [answered]
i am making a BBS plugin, Using DiscordSRV, EssentialsX, PlaceholderAPI and More.
but how can i add DiscordSRV API as a Maven Dependency?

**Comment by Dinty1** [comment, marked as answer]
See https://docs.discordsrv.com/#developers for details + example
